script.js

```diff
@@ -44,6 +44,21 @@ idApp.eventListeners = function() {
       }
    })
 
+   // Event Listener #3A for when any radio input is selected to display a div with some more info
+   $main.on('click', '.optionContainer', function () {
+      $('.hiddenContents').removeClass('displayContents');
+      $('.fa-plus').removeClass('active')
+      $(this).find('.hiddenContents').toggleClass('displayContents');
```

Comment:
Awesome that you were able to figure out how to do $(this)! (see what I did there) 
Cool to learn what .find() does...will def keep that in my pocket for future projects!